fe/fe-core/src/main/java/com/starrocks/sql/optimizer/SubqueryUtils.java

```diff
@@ -101,7 +99,7 @@ public static Pair<ScalarOperator, OptExprBuilder> rewriteScalarOperator(
     public static LogicalPlan getLogicalPlan(ConnectContext session, CTETransformerContext cteContext,
                                              ColumnRefFactory columnRefFactory, QueryRelation relation,
                                              ExpressionMapping outer) {
-        if (!(relation instanceof SelectRelation) && !(relation instanceof SubqueryRelation)) {
+        if (!(relation instanceof QueryRelation)) {
             throw new SemanticException("Currently only subquery of the Select type are supported");
```

Comment:
do we need update the exception message at the same time?

Reply:
I think it's OK. Query relation means that this is a query statement. It can be a unino or except query.